test/unit/math/rev/mat/fun/cholesky_decompose_test.cpp

```diff
@@ -226,7 +230,7 @@ void test_gp_grad(int mat_size, double prec) {
   Eigen::VectorXd test_vals(3);
   test_vals[0] = 1;
   test_vals[1] = 1.5;
-  test_vals[2] = 1;
+  test_vals[2] = 10;
```

Comment:
test_vals[2] is the jitter that is added to the diagonal. We up that a bit so that large matrices are more stable

Reply:
@rok-cesnovar didn't eigen also fail for large N here?

Reply:
Yep, it seems that this way of producing the input matrix is not suitable for large matrices.